templates/pages/management/document_item.html.twig

```diff
@@ -0,0 +1,98 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2024 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% import 'components/form/basic_inputs_macros.html.twig' as inputs %}
+
+<div class="mb-3">
+    <form name="documentitem_form{{ rand }}" id="documentitem_form{{ rand }}" method="post"
+          action="{{ 'Document'|itemtype_form_path }}" enctype="multipart/form-data" data-submit-once>
+
+        <div class="d-flex">
+            {{ fields.dropdownField('DocumentCategory', 'documentcategories_id', item.fields['documentcategories_id'], 'DocumentCategory'|itemtype_name, {
+                field_class: 'col-12 col-sm-5 pe-2',
+                entity: entities
+            }) }}
+            {{ fields.fileField('filename', null, null, {
+                no_label: true,
+                field_class: 'col-12 col-sm-5 pe-2',
+                multiple: true
+            }) }}
+            {% set btn %}
+                {{ inputs.submit('add', _x('button', 'Add a new file'), 1) }}
+            {% endset %}
+            {{ fields.htmlField('', btn, null, {
+                field_class: 'col-12 col-sm-2 text-end align-self-center',
+                no_label: true
+            }) }}
+
+            {{ inputs.hidden('entities_id', entity) }}
+            {{ inputs.hidden('is_recursive', item.isRecursive) }}
+            {{ inputs.hidden('itemtype', item.getType) }}
+            {{ inputs.hidden('items_id', item.getID) }}
+            {% if item.getType == 'Ticket' %}
+                {{ inputs.hidden('tickets_id', item.getID) }}
```

Comment:
Please add the `()` after the methods names, it is easier to understand that we rely on the class methods rather than on its properties, and it ensures that the occurence will be found if we search for `xxx()`.
```suggestion
            {{ inputs.hidden('is_recursive', item.isRecursive()) }}
            {{ inputs.hidden('itemtype', item.getType()) }}
            {{ inputs.hidden('items_id', item.getID()) }}
            {% if item.getType() == 'Ticket' %}
                {{ inputs.hidden('tickets_id', item.getID()) }}
```